etl/transformer/transformer.go

```diff
@@ -26,26 +26,23 @@ import (
 	"github.com/kelindar/binary"
 )
 
-const (
-	// delta between pulses
-	pulseDelta uint16 = 10
-)
-
 // Transform transforms thr row JetDrops to canonical JetDrops
 func Transform(ctx context.Context, jd *types.PlatformJetDrops) ([]*types.JetDrop, error) {
-	// if no records per pulse
-	if len(jd.Records) == 0 {
-		return make([]*types.JetDrop, 0), nil
-	}
+	pulseData := getPulseData(jd.Pulse)
 
-	pulseData, err := getPulseData(jd.Records[0])
+	m, err := getRecords(jd)
 	if err != nil {
-		return nil, errors.Wrapf(err, "cannot get pulse data from record")
+		return nil, err
 	}
 
-	m, err := getRecords(jd.Records)
-	if err != nil {
-		return nil, err
+	log := belogger.FromContext(ctx)
```

Comment:
and could you pls add the logger field here? it will help us to understand who logged

Reply:
please suggest a code how you want to see it